### PR TITLE
MAHOUT-1653 Spark-1.3 for 0.10.0

### PR DESCRIPTION
**Pull request by andrewpalumbo**
Uses a  deprecated method in `CheckpointedDrmSpark.scala` and has the shell build disabled.

### CODE REVIEW

File: spark/src/main/scala/org/apache/mahout/sparkbindings/drm/CheckpointedDrmSpark.scala [hunk 1]
```diff
@@ -165,7 +167,8 @@ class CheckpointedDrmSpark[K: ClassTag](
       else if (classOf[Writable].isAssignableFrom(ktag.runtimeClass)) (x: K) => x.asInstanceOf[Writable]
       else throw new IllegalArgumentException("Do not know how to convert class tag %s to Writable.".format(ktag))
 
-    rdd.saveAsSequenceFile(path)
+    // this is a (working) deprecated method used as a stop-gap while we investigate the shell issues
```
Comment (dlyubimov) [inline spark/src/main/scala/org/apache/mahout/sparkbindings/drm/CheckpointedDrmSpark.scala]: ok stop-gap but perhaps we can do it on top of #135 ..

Comment (andrewpalumbo) [inline spark/src/main/scala/org/apache/mahout/sparkbindings/drm/CheckpointedDrmSpark.scala]: yeah- this definitely can wait..  I'll re-synch it after #135 is pushed and we can keep it here as a patch for people that want to use 1.3 on the 0.10.x branch.

**Comment by pferrel** [comment]
This should be closed in favor of #136 so it goes into 0.11.0

**Comment by andrewpalumbo** [comment]
see #136

**Pull request closed without merging**